site/src/modules/dashboard/Navbar/UserDropdown/UserDropdownContent.tsx

```diff
@@ -109,7 +85,8 @@ export const UserDropdownContent: FC<UserDropdownContentProps> = ({
 						>
 							<PopoverClose asChild>
 								<MenuItem css={styles.menuItem}>
-									{renderMenuIcon(link.icon)}
+									{link.icon !== "" &&
```

Comment:
```suggestion
									{link.icon &&
```

empty string coerces to `false`, and this also handles potential cases like `undefined`